js/app/modules/contentGroup/contentGroup.js

```diff
@@ -217,6 +227,30 @@ const ContentGroup = new Module.Class({
         if (!this._selection.in_error_state)
             return;
         this._stack.visible_child_name = ERROR_PAGE_NAME;
+        this._log_button_stack.visible_child_name = 'button';
+    },
+
+    _on_log_button_click: function () {
+        let timestamp = new Date(Date.now());
+        let stamp = 'eos-knowledge-lib log ' + timestamp.toISOString() + '.txt';
+        let log_name = GLib.build_filenamev([GLib.get_home_dir(), stamp]);
```

Comment:
a) On the other hand, perhaps it would be better to not ask for an extra permission, instead just open the log file in gedit so the user can copy/paste it and not care about its location being discoverable. For now I'll open a pull request adding the permission to the build manifest but let me know what you think.